docs/source/federation-versions.mdx

```diff
@@ -0,0 +1,565 @@
+---
+title: Federation version changelog
+description: Understand changes between federation versions
+---
+
+## v2.3
+
+**Directive changes**
+
+
+<table>
+  <thead>
+    <tr>
+      <th>Name</th>
+      <th>Description</th>
+    </tr>
+  </thead>
+
+<tbody>
+<tr>
+<td>
+
+##### `@interfaceObject`
+
+</td>
+<td>
+
+```graphql
+directive @interfaceObject on OBJECT
+```
+
+</td>
+</tr>
+
+</tbody>
+</table>
+
+**Subgraph changes**
+
+
+<table>
+  <thead>
+    <tr>
+      <th>Name</th>
+      <th>Description</th>
+    </tr>
+  </thead>
+
+<tbody>
+<tr>
+<td>
+
+Interfaces
+
+</td>
+<td>
+
+`@key` directive can now be used on interface definitions.
+
+</td>
+</tr>
+
+</tbody>
+</table>
+
+
+## v2.2
+
+**Directive changes**
+
+
+<table>
+  <thead>
+    <tr>
+      <th>Name</th>
+      <th>Description</th>
+    </tr>
+  </thead>
+
+<tbody>
+<tr>
+<td>
+
+##### `@shareable`
+
+</td>
+<td>
+
+Added `repeatable` to the directive definition.
+
+```graphql
+directive @shareable repeatable on OBJECT | FIELD_DEFINITION
+```
+
+</td>
+</tr>
+
+</tbody>
+</table>
+
+**Subgraph changes**
+
+
+<table>
+  <thead>
+    <tr>
+      <th>Name</th>
+      <th>Description</th>
+    </tr>
+  </thead>
+
+<tbody>
+<tr>
+<td>
+
+Interfaces
+
+</td>
+<td>
+
+Composition now rejects `@shareable` on interface fields.
```

Comment:
I might also fold this info into the table entry above for `@shareable` so it says:

> Added `repeatable` to the directive definition. Additionally, composition now rejects `@shareable` when applied to interface fields.